test/correctness/vectorize_nested.cpp

```diff
@@ -0,0 +1,261 @@
+#include "Halide.h"
+#include "test/common/check_call_graphs.h"
+
+using namespace Halide;
+
+int vectorize_2d_round_up() {
+    const int width = 32;
+    const int height = 24;
+
+    Func f("f");
+    Var x("x"), y("y"), xi("xi"), yi("yi");
+
+    f(x, y) = x + y;
```

Comment:
It might be good to make this function x + 3 * y or something so bugs that swap x and y are visible to the test (same below).

Reply:
Done